### DISCUSSION TITLE
Disponibilizar o conteúdo via Medium

### DISCUSSION THREAD

**Discussion by GuilhermeCaruso** [vtbarreto/go-fundamentals#1] [Q&A] [answered]
Por se tratar de "todos" os fundamento da linguagem, não pensa em organizar este repositório de maneira a construir um material mais robusto utilizando posts no medium. Explicando cada step do processo de estudo?

**Comment by vtbarreto** [comment, marked as answer]
Estou trabalhando na criação desse artigo e em breve vou disponibiza-lo
